pkg/apb/client.go

```diff
@@ -227,3 +228,21 @@ func OcLogin(log *logging.Logger, args ...string) error {
 	log.Debug(string(output))
 	return nil
 }
+
+// TODO: Instead of putting namespace directly as a parameter, we should create a dictionary
+// of apb_metadata and put context and other variables in it so we don't pollute the user
+// parameter space.
+func createExtraVars(context *Context, parameters *Parameters) (string, error) {
+	var paramsCopy Parameters
+	if parameters != nil && *parameters != nil {
```

Comment:
Unfortunately, I ran into the case of "bind" in testing where the parameters wasn't nil, but was pointing to an uninitialized map.  in this case `parameters != nil` but `*parameters == nil`.

Jesus pointed out that if nil could be passed in as the `parameters` argument as well as he showed in his unit test, so I should also check for that.